README.md

```diff
@@ -204,6 +260,10 @@ resolves the returned promise.
 - **openBox**(<*string*> boxName, [<*function*> callback]) - *Promise* - Open a mailbox, calling the provided callback
 with signature `(err, boxName)`, or resolves the returned promise with `boxName`.
 
+- **closeBox**(<*boolean*> autoExpunge, [<*function*> callback]) - *Promise* - Close a mailbox, calling the provided callback
```

Comment:
putting brackets around the autoExpunge here will help users know it's not necessary to provide a value each time

`**closeBox**([<*boolean*> autoExpunge = true], [<*function*> callback])`